tools/setup_env.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash -e
-gcloud components install app-engine-java
-
-cd ui
-echo Installing the Angular command-line tool, to serve/build the UI.
-sudo npm install -g @angular/cli
-echo Installing UI project dependencies.
-npm install
-
-
-cd ../tools
-./download_swagger_cli.sh
-./download_cloud_sql_proxy.sh
```

Comment:
I don't believe we are using this right now, but I'm going to leave the file here since I'm doing a pass on the API next to fix the MySQL connection.